src/mbgl/map/source.cpp

```diff
@@ -521,11 +521,16 @@ bool Source::update(MapData& data,
     return allTilesUpdated;
 }
 
-void Source::invalidateTiles(const std::vector<TileID>& ids) {
+void Source::invalidateTiles(const std::unordered_set<TileID, TileID::Hash>& ids) {
```

Comment:
Can you explain why this is an unordered set? TileIDs should be sortable fairly easily without having to resort to hashing them.